# config/detector.yaml

detector:
  weights: "$(find perception_test)/models/yolo11m.pt"
  img_size: 640
  conf_thres: 0.25                            
  iou_thres: 0.45
  device: "cuda:0"
  classes: [2, 3, 5, 7] # car, motorcycle, bus, truck

runtime:
  half: true
  agnostic_nms: true  # [수정] false -> true (클래스 무관하게 겹치면 제거)
  verbose: false

topics:
  image_in: /camera/image_raw
  detections_out: /perception_test/detections_raw # 노드 이름과 맞춤